src/pages/Access.tsx

```diff
@@ -36,17 +37,40 @@ function extractSwarmHash(string: string): string | null {
   return (matches && matches[0]) || null
 }
 
-function recognizeSwarmHash(value: string) {
-  if (value.length < 64) {
-    return value
+function extractSwarmCid(s: string): string | null {
+  const matches = s.match(/bah[a-z0-9]{58}/)
```

Comment:
Don't use this sort of "type assertion" please. Leave it up to the CID library to decide if it is a valid CID or not. I would remove this check. Btw. it is also already redundant as the library is doing similar check.

Reply:
This is for matching only the CID part if a url is pasted.

Reply:
Hmm can't we have it then parsed like `*.bzz.link`? I would like to avoid hardcoding the form of the CID...

Reply:
I changed the logic, but since the `bzz.link` is a configurable value I wanted to avoid hardcoding that either.